index.html

```diff
@@ -799,6 +802,38 @@ <h2>
           <li>Let |acquisitionTime:EpochTimeStamp| be a new {{EpochTimeStamp}}
           that represents now.
           </li>
+          <li>
+            <ins cite="#a4"> Check if the emulated position data should be used by running
```

Comment:
This check should be after we check that the site has permission to access geolocation, but probably before we check for a cached position, since when emulation is enabled we no longer want to return a non-emulated value that might still be present in the cache.

Reply:
agree. Moved.